docs/minikube.md

```diff
@@ -1,75 +1,93 @@
-### Requirements
+# Deploying and Configuring Minikube
+
+## Requirements
 - [kubectl](https://kubernetes.io/docs/tasks/tools/install-kubectl/)
-- [minikube](https://github.com/kubernetes/minikube/#installation) v0.25
-  - [virtualbox](https://www.virtualbox.org/wiki/Downloads)
-- [helm](https://github.com/kubernetes/helm#install) v2.8.2
+- [minikube](https://github.com/kubernetes/minikube/#installation) v0.5
```

Comment:
Lets just do the latest and greatest. `v1.0.1` this is what I tested out on today.

Reply:
done